Src/AmrCore/AMReX_AmrMesh.H

```diff
@@ -33,6 +33,10 @@ struct AmrInfo {
     bool use_fixed_coarse_grids = false;
     //chop up grids to have the number of grids no less the number of procs
     bool refine_grid_layout = true;
+
+    /** Allow grids to be refined in only the dimensions */
+    std::vector<int> refine_grid_layout_dims = { 1, 1, 1};
```

Comment:
I suggest we use `IntVect` instead of `vector`.  Then it's clear from its type that it has AMREX_SPACEDIM elements, and it's for spatial dimensions, not amr levels.

Reply:
Yes. I was going to ask about this. I was having issues with reading in the `IntVect` with ParamParse. For line 38, I was trying:
```
IntVect refine_grid_layout_dims = {1, 1, 1};
```
or
```
Vector<int> refine_grid_layout_dims = {1,1,1}
```
But I couldn't get these to compile. Can you tell me what it should it be? Then I can fix the rest of the code.

Reply:
I think you can do `IntVect refine_grid_layout_dims = IntVect(1);`.